src/components/alert/InfoAlert.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Alert, AlertProps } from './Alert';
+import { useTheme } from '../../providers';
+import { InfoOutlined } from '../icons';
+
+export interface InfoAlertProps extends AlertProps {}
+
+export const InfoAlert = ({ ...rest }: InfoAlertProps) => {
+  const {
+    theme: {
+      current: {
+        customColors: { blue1 },
+      },
+    },
+  } = useTheme();
+  return <Alert {...rest} color={blue1} startIcon={<InfoOutlined size="20" />} />;
```

Comment:
@hiddeottens maybe it'll be better to replace "20" with `scale700`?

Reply:
yea you're right